src/Core/Domain/Order/Exception/InvalidOrderStateException.php

```diff
@@ -33,14 +33,14 @@
 class InvalidOrderStateException extends OrderException
 {
     /**
-     * Used when the order has no invoice (and it should have)
+     * Used when the order is not paid (and it should be)
      */
-    const INVOICE_NOT_FOUND = 1;
+    const NOT_PAID = 1;
 
     /**
-     * Used when the order has an invoice (and it should not)
+     * Used when the order is already paid (and it should not be)
      */
-    const UNEXPECTED_INVOICE = 2;
+    const ALREADY_PAID = 2;
```

Comment:
BC may break.

Reply:
It's not a BC... This class and thoses constants don't exist in 176x.

Reply:
Indeed, this class has been introduced in 177, so we can still modify it without BC break for now